harness/determined/cli/agent.py

```diff
@@ -79,16 +76,16 @@ def list_slots(args: argparse.Namespace) -> None:
         if r["container_id"]
     }
 
-    def device_type_string(deviceType: typing.Optional[devicev1Type]) -> str:
-        if deviceType == devicev1Type.CUDA:
+    def device_type_string(deviceType: typing.Optional[bindings.devicev1Type]) -> str:
```

Comment:
just a thought: IMO for some very common modules it might make sense to break the rule and use shorter aliases (eg how they do `numpy as np` or pandas as pd) but then it might not be clear to everyone and this loses its value.

`bindings` or `typing` would be good candidates.

Reply:
`typing` is already excepted from this rule. adding our own custom exception for anything else would mean a fork.

bindings is not the worst thing. for example, in CLI `from determined.common import declarative_argparse` looks pretty bad. agreeing on a standardized import alias, e.g. `from determined.common import declarative_argparse as dap` would be nice.